backend/src/main/java/de/neuefische/backend/service/UserRegistrationService.java

```diff
@@ -0,0 +1,42 @@
+package de.neuefische.backend.service;
+
+import de.neuefische.backend.BackendApplication;
+import de.neuefische.backend.model.RegistrationData;
+import de.neuefische.backend.model.User;
+import de.neuefische.backend.repository.MongoUserRepository;
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DuplicateKeyException;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.stereotype.Service;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.List;
+
+@Service
+public class UserRegistrationService {
+
+    Log LOG = LogFactory.getLog(BackendApplication.class);
+
+    @Autowired
+    MongoUserRepository repository;
+
+    @Autowired
+    PasswordEncoder encoder;
+
+    public String register(RegistrationData registerUser) {
+        LOG.info(registerUser.getName()+ " " +registerUser.getPassword());
```

Comment:
DONT log the password.

Reply:
My fault: I forgot to delete it after the test.